src/flight-search/filtering.js

```diff
@@ -5,6 +5,19 @@ function filterByPrice(trip, priceRange) {
   return trip.fares[0] && utils.filterByRange(trip.fares[0].price.amountUsd, priceRange);
 }
 
+function filterByFlexibleTickets(trip, flexible = []) {
+  const isRefundableFlag = flexible.map(str => str.toLowerCase()).includes("refundable");
```

Comment:
can just do `flexible.includes("refundable")` (i'm assuming the value is already lowercased tho)

Reply:
updated